File: example_problems/tutorial/bit_edit_to_zero/old/att/template_sol.c
#include"assert.h"
#include"stdio.h"

//##########################################
// BEGIN: modifica entro quest'area

int num_mosse(int n) {
    return 43;
}

int mossa(int n) {
  assert(n > 0);
  return 1;
}

// END: modifica entro quest'area
// ################################################


int main() {
  int p, n;
  scanf("%d %d",&p, &n);
  assert(p >= 1);
  assert(p <= 2);
  assert(n >= 0);
  if(p  == 1)
    printf("%d\n",num_mosse(n));
  if(p  == 2)
    printf("%d\n",mossa(n));

  return 0;
}

File: example_problems/tutorial/bit_edit_to_zero/old/att/template_sol.py
#!/usr/bin/env python3
# -*- coding: utf-8 -*-
# Soluzione di bit_edit_to_zero

from __future__ import print_function
import sys

if sys.version_info < (3, 0):
    input = raw_input # in python2, l'equivalente di input è raw_input

#################################################
# BEGIN: parte di tua competenza

def num_mosse(n):
    return 43

def mossa(n):
    assert n > 0
    return 1

# END: parte di tua competenza
#################################################
    
    
p, n = map(int, input().strip().split())
assert 1 <= p <= 2
assert n >= 0
if p  == 1:
    print(num_mosse(n))
if p  == 2:
    print(mossa(n))
    
